packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -375,7 +376,7 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     scrollbarWidth: 0,
   });
 
-  const valueToIndex = React.useRef(new Map()).current;
+  const valueToIndex = useLazyRef(() => new Map()).current;
```

Comment:
`() => new Map()` can be extracted to a function so a new one isn't instantiated every time this code runs.

Reply:
Done in https://github.com/mui/material-ui/pull/46333/commits/acc42c28a2a484209f2196d11bb5e5e51a9e5948